docs/pages/access-controls/access-requests/resource-requests.mdx

```diff
@@ -133,8 +133,33 @@ To request access to these resources, run
     --reason <request reason>
 ```
 
-You can search for resources of kind `node`, `kube_cluster`, `pod`, `db`, `app`, and
-`windows_desktop`. Advanced filters and queries are supported. See our
+You can search for resources of kind `node`, `kube_cluster`, `db`, `app`, and
+`windows_desktop`. Teleport also supports searching and requesting access to
+resources within Kubernetes clusters, such as:
+
+- `pod`: Grant acess to individual pods.
```

Comment:
I think we should avoid repeating "Grant access to individual..." in each item. Actually, since each term lines up with a Kubernetes resource, I think it's okay to list each one without explanation.

I would add this list to a `Details` box so it doesn't take up as much room on this page.

Reply:
236fbd2b919631466e288b1f3a408e18dce5b000